yarn_api_client/resource_manager.py

```diff
@@ -203,7 +217,7 @@ def cluster_application_statistics(self, states=None,
         state and the number of applications of this type and this state in
         ResourceManager context.
 
-        This method work in Hadoop > 2.0.0
+        This method only works in Hadoop > 2.0.0
```

Comment:
Just curious if this should really be `Hadoop >= 2.0.0`?  Seems odd that this would only work on versions of Hadoop >= 2.0.1.

Reply:
Honestly, I even lost trace when this was added.

I checked in 2.0.0-alpha, and this api is not mentioned
https://github.com/apache/hadoop/blob/da30cf664ccc99a54059eb6b9ffa73dc68a95ed2/hadoop-mapreduce-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/ResourceManagerRest.apt.vm

Not quite sure when it was exactly added, but definitely long ago.
Closest PR with respective change I saw was Apache's documentation correction ticket.
https://issues.apache.org/jira/browse/HADOOP-14364

Reply:
OK - not a big deal since the version reference was already there.